File: q3.java
// Find the max and min element in an array
public class q3 {
    public static void main(String[] args) {
        int [] arr = {19,28,293,823,93};

        int mx = Integer.MIN_VALUE;
        int mn = Integer.MAX_VALUE;
        for(int i = 0; i < arr.length; i++){
            mx = Math.max(mx, arr[i]);
            mn = Math.min(mn, arr[i]);
        }

        System.out.println(mx);
        System.out.println(mn);
    }
}

File: q3.cpp
// Find the max and min element in an array
#include<bits/stdc++.h>
using namespace std;

int main() {
    int ar [] = {23,232, 43, 2921, 873, 982};
    int mx = INT_MIN;
    int mn = INT_MAX;

    for(int i = 0; i < 6; i++){
        mx = max(ar[i], mx);
        mn = min(ar[i], mn);    
    }

    cout<< mx << endl << mn;
    return 0;
}